UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -828,8 +846,333 @@ public IEnumerator ProcessDNAPayload(DNAMutationData InDNAMutationData)
+			{
+				if (x.NamedSlot != null && !bodyParts.ContainsKey((NamedSlot)x.NamedSlot))
+					bodyParts.Add((NamedSlot)x.NamedSlot, x);
+			}
 
+			foreach (var itemForPlace in itemsBeforeTransform)
+			{
+				if (itemForPlace.Item1 != null && bodyParts.ContainsKey(itemForPlace.Item2))
+				{
+					Inventory.ServerAdd(itemForPlace.Item1.gameObject, bodyParts[itemForPlace.Item2]);
+				}
+			}
+
+			if (dna != null && changeling != null)
+			{
+				var storage = changeling.ChangelingMind.CurrentPlayScript.DynamicItemStorage;
+
+				foreach (var id in dna.BodyClothesPrefabID)
+				{
+					var fakeClothes = Spawn.ServerPrefab(CustomNetworkManager.Instance.ForeverIDLookupSpawnablePrefabs[id]).GameObject;
+
+					ItemSlot bestSlot = storage.GetBestSlotFor(fakeClothes);
+
+					var placed = Inventory.ServerAdd(fakeClothes, bestSlot);
+
+					if (placed && (bestSlot.NamedSlot == NamedSlot.storage01 || bestSlot.NamedSlot == NamedSlot.storage02 || bestSlot.NamedSlot == NamedSlot.storage03
+					 || bestSlot.NamedSlot == NamedSlot.storage04 || bestSlot.NamedSlot == NamedSlot.storage05 || bestSlot.NamedSlot == NamedSlot.storage06
+					  || bestSlot.NamedSlot == NamedSlot.storage07 || bestSlot.NamedSlot == NamedSlot.storage08 || bestSlot.NamedSlot == NamedSlot.storage09
+					   || bestSlot.NamedSlot == NamedSlot.storage10 || bestSlot.NamedSlot == NamedSlot.storage11 || bestSlot.NamedSlot == NamedSlot.storage12
+						|| bestSlot.NamedSlot == NamedSlot.storage13 || bestSlot.NamedSlot == NamedSlot.storage14)) // better don`t put fake clothes into storages
+					{
+						placed = false;
+					}
+
+
+
+					var itemName = fakeClothes.GetComponent<ItemAttributesV2>().InitialName;
+
+					if (!placed)
+					{
+						_ = Despawn.ServerSingle(fakeClothes);
+						continue;
+					}
+					//making items absolutely useless
+					fakeClothes.GetComponent<ItemAttributesV2>().IsFakeItem = true;
+
+					if (fakeClothes.TryGetComponent<WearableArmor>(out var armor))
+					{
+						foreach (var bodyArmr in armor.ArmoredBodyParts)
+						{
+							bodyArmr.Armor.Melee = 0;
+							bodyArmr.Armor.Bullet = 0;
+							bodyArmr.Armor.Laser = 0;
+							bodyArmr.Armor.Energy = 0;
+							bodyArmr.Armor.Bomb = 0;
+							bodyArmr.Armor.Rad = 0;
+							bodyArmr.Armor.Fire = 0;
+							bodyArmr.Armor.Acid = 0;
+							bodyArmr.Armor.Magic = 0;
+							bodyArmr.Armor.Bio = 0;
+							bodyArmr.Armor.Anomaly = 0;
+							bodyArmr.Armor.DismembermentProtectionChance = 0;
+							bodyArmr.Armor.StunImmunity = false;
+							bodyArmr.Armor.TemperatureProtectionInK = new Vector2(283.15f, 283.15f + 20);
+							bodyArmr.Armor.PressureProtectionInKpa = new Vector2(30f, 300f);
+
+						}
+					}
+					if (fakeClothes.TryGetComponent<IDCard>(out var card))
+					{
+						var cardJobName = card.GetJobTitle();
+						var cardRegName = card.RegisteredName;
+						card.ServerChangeOccupation(OccupationList.Instance.Get(JobType.ASSISTANT), true, true);
+
+						//card.SyncJobTitle("", cardName);
```

Comment:
don't leave in comments unless you leave a note explaining why they must be left behind. It will anger our overlord, Codacy.

Reply:
Gonna try my best for no anger the overlord